tests/MongoDB.Driver.Core.Tests/Core/WireProtocol/CommandWriteProtocolTests.cs

```diff
@@ -91,7 +91,8 @@ public void Execute_should_use_cached_IWireProtocol_if_available([Values(false,
             else
             {
                 var e = exception.Should().BeOfType<ArgumentException>().Subject;
-                e.Message.Should().Be("CommandResponseHandling must be Return, NoneExpected or ExhaustAllowed.\r\nParameter name: responseHandling");
+                e.Message.Should().StartWith("CommandResponseHandling must be Return, NoneExpected or ExhaustAllowed.");
```

Comment:
NET Core 3 uses a bit different format in the final message regarding including `ParamName` information. So splitting it into two validations solved this issue